packages/@aws-cdk/aws-cloudwatch/lib/private/metric-util.ts

```diff
@@ -0,0 +1,109 @@
+import { Duration } from "@aws-cdk/core";
+import { MathExpression } from "../metric";
+import { IMetric, MetricConfig, MetricExpressionConfig, MetricStatConfig } from "../metric-types";
+
+export function validateNoIdConflicts(expression: MathExpression) {
```

Comment:
Can this just be a private method on `MathExpression`? It doesn't look like it's used anywhere else but that class.